src/dotnet/APIView/APIViewUnitTests/APIViewUnitTests.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.10.0" />
```

Comment:
I love this library and encourage fluent style tests.

Reply:
Which I think should happen, otherwise it feels like a little much to bring in a new dependency for a single unit test.

Reply:
Ok, makes sense I can probably just have the languages to be always sorted.